src/pages/SwapV3/ApprovalModal.tsx

```diff
@@ -0,0 +1,253 @@
+                <path
+                  fillRule="evenodd"
+                  clipRule="evenodd"
+                  d="M26.6384 19.305C27.1902 18.755 27.7659 18.1775 28.3434 17.6C30.7707 15.1727 34.7069 15.1727 37.1342 17.6C37.1342 17.6018 37.136 17.6037 37.1379 17.6055C39.5652 20.0328 39.5652 23.9672 37.1379 26.3945C37.136 26.3963 37.1342 26.3982 37.1342 26.4C34.7069 28.8273 30.7707 28.8273 28.3434 26.4C25.6594 23.7142 22.9717 21.0283 22.9717 21.0283C22.4364 20.4912 21.5637 20.4912 21.0284 21.0283C20.4912 21.5637 20.4912 22.4363 21.0284 22.9717C21.0284 22.9717 23.7142 25.6593 26.4 28.3433C29.8999 31.845 35.5777 31.845 39.0775 28.3433C39.0794 28.3415 39.0812 28.3415 39.083 28.3397C42.5829 24.838 42.5829 19.162 39.083 15.6603C39.0812 15.6585 39.0794 15.6585 39.0775 15.6567C35.5777 12.155 29.8999 12.155 26.4 15.6567C25.8225 16.2342 25.245 16.8098 24.695 17.3617C24.1579 17.897 24.1579 18.7697 24.695 19.305C25.2304 19.8422 26.103 19.8422 26.6384 19.305Z"
+                  fill="currentcolor"
+                  filter="blur(2px)"
+                />
+              </svg>
+            </RowFit>
+            <MouseoverTooltip
+              text={t`You wish to give KyberSwap permission to use the selected token for transactions without any limit. You do not need to give permission again unless you have revoked it. This approve transaction will cost a gas fee`}
+              placement="right"
+            >
+              <Text
+                fontSize="16px"
+                lineHeight="20px"
+                fontWeight={500}
+                style={{ textDecoration: 'underline 1px dotted', textUnderlineOffset: '4px' }}
+              >
+                <Trans>Infinite Allowance</Trans>
+              </Text>
+            </MouseoverTooltip>
+          </OptionWrapper>
+
+          <OptionWrapper active={option === ApproveOptions.Custom} onClick={() => setOption(ApproveOptions.Custom)}>
+            <RowFit flex="0 0 48px">
+              <svg width="44" height="44" viewBox="0 0 44 44" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <g clipPath="url(#clip0_398_6043)">
+                  <path
+                    d="M5.5 31.6249V38.4999H12.375L32.6517 18.2233L25.7767 11.3483L5.5 31.6249ZM37.9683 12.9066C38.6833 12.1916 38.6833 11.0366 37.9683 10.3216L33.6783 6.03161C32.9633 5.31661 31.8083 5.31661 31.0933 6.03161L27.7383 9.38661L34.6133 16.2616L37.9683 12.9066Z"
+                    fill="currentcolor"
+                  />
+                </g>
+                <g clipPath="url(#clip1_398_6043)" filter="blur(2px)">
+                  <path
+                    d="M5.5 31.6249V38.4999H12.375L32.6517 18.2233L25.7767 11.3483L5.5 31.6249ZM37.9683 12.9066C38.6833 12.1916 38.6833 11.0366 37.9683 10.3216L33.6783 6.03161C32.9633 5.31661 31.8083 5.31661 31.0933 6.03161L27.7383 9.38661L34.6133 16.2616L37.9683 12.9066Z"
+                    fill="currentcolor"
+                  />
+                </g>
+              </svg>
```

Comment:
Can we use react feather?
https://feathericons.com/?query=edit

Reply:
its different one, there is needed to blur the svg 
![image](https://user-images.githubusercontent.com/9592090/233294973-22a42a00-a7d9-44a9-a3f3-aea3fc4b991f.png)